chart/openfaas/README.md

```diff
@@ -219,6 +219,23 @@ By default services will be exposed with following hostnames (can be changed, se
 
 If you require TLS/SSL then please make use of an IngressController. A full guide is provided to [enable TLS for the OpenFaaS Gateway using cert-manager and Let's Encrypt](https://docs.openfaas.com/reference/ssl/kubernetes-with-cert-manager/).
 
+### Istio mTLS
+
+To install OpenFaaS with Istio mTLS pass  `--set istio.mtls=true` and disable the HTTP probes:
```

Comment:
I think you mentioned this forgot, sorry (I forgot) but are httpProbes available for newer versions of Istio, or not available for any?

Reply:
I've tried to enable those on Istio 1.2.3 and it didn't work.

Reply:
I'll merge as it, but can you look into this when you have time?